test/skimNtuple2017_HHbtag.cpp

```diff
@@ -3766,50 +3766,51 @@ int main (int argc, char** argv)
     for (unsigned int iLep = 0 ; (iLep < theBigTree.daughters_px->size ()) ; ++iLep)
     {
       // skip the H decay candiates
-      if (int (iLep) == firstDaughterIndex || int (iLep) == secondDaughterIndex) continue;
+      if (int (iLep) == firstDaughterIndex || int (iLep) == secondDaughterIndex)
+		continue;
 
       // remove taus
       if (theBigTree.particleType->at (iLep) == 2)
-      {
-	continue ;
-      }
+		{
+		  continue ;
+		}
       else if (theBigTree.particleType->at (iLep) == 0) // muons
-      {
-	// Fra Mar2020: for muon, Tight does not imply Medium so we check both
-	bool passMed = oph.muBaseline (&theBigTree, iLep, 10., 2.4, 0.3, OfflineProducerHelper::MuMedium);
-	bool passTig = oph.muBaseline (&theBigTree, iLep, 10., 2.4, 0.3, OfflineProducerHelper::MuTight);
-	if (!passMed && !passTig) continue; // if it passes one of the two --> the "if" is false and the lepton is saved as an extra lepton
-      }
+		{
+		  // Fra Mar2020: for muon, Tight does not imply Medium so we check both
+		  bool passMed = oph.muBaseline (&theBigTree, iLep, 10., 2.4, 0.3, OfflineProducerHelper::MuMedium);
+		  bool passTig = oph.muBaseline (&theBigTree, iLep, 10., 2.4, 0.3, OfflineProducerHelper::MuTight);
+		  if (!passMed && !passTig) continue; // if it passes one of the two --> the "if" is false and the lepton is saved as an extra lepton
+		}
       else if (theBigTree.particleType->at (iLep) == 1) // electrons
-      {
-	// Fra Mar2020: for electron, we check (mvaEleID-Fall17-iso-V2-wp90 OR (mvaEleID-Fall17-noIso-V2-wp90 AND pfRelIso < 0.3))
-	if (DEBUG) std::cout << "--- Debug for extra electrons:" << std::endl;
-	bool passIso    = oph.eleBaseline (&theBigTree, iLep, 10., 2.5, 0.3, OfflineProducerHelper::EMVAMedium, string("Vertex-LepID-pTMin-etaMax"), (DEBUG ? true : false));
-	bool passNonISo = oph.eleBaseline (&theBigTree, iLep, 10., 2.5, 0.3, OfflineProducerHelper::EMVAMedium, string("Vertex-pTMin-etaMax-thirdLep"), (DEBUG ? true : false));
-	if (!passIso && !passNonISo) continue; // if it passes one of the two --> the "if" is false and the lepton is saved as an extra lepton
-      }
+		{
+		  // Fra Mar2020: for electron, we check (mvaEleID-Fall17-iso-V2-wp90 OR (mvaEleID-Fall17-noIso-V2-wp90 AND pfRelIso < 0.3))
+		  if (DEBUG) std::cout << "--- Debug for extra electrons:" << std::endl;
+		  bool passIsoMVA = oph.eleBaseline (&theBigTree, iLep, 10., 2.5, 0.3, OfflineProducerHelper::EMVAMedium, string("Vertex-LepID-pTMin-etaMax"), (DEBUG ? true : false));
+		  //bool passNonIsoMVA = oph.eleBaseline (&theBigTree, iLep, 10., 2.5, 0.3, OfflineProducerHelper::EMVAMedium, string("Vertex-pTMin-etaMax-thirdLep"), (DEBUG ? true : false));
+		  if (!passIso) continue; // if it passes --> the "if" is false and the lepton is saved as an extra lepton
```

Comment:
Should be `if (!passIsoMVA) continue;` ?

Reply:
also this change was not applied to the 2016 skimmer, is that on purpose?

Reply:
not at all; thanks for noticing